crypto/src/main/java/net/consensys/cava/crypto/mikuli/KeyPair.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package net.consensys.cava.crypto.mikuli;
+
+public final class KeyPair {
+
+  private final PrivateKey privateKey;
+  private final PublicKey publicKey;
+
+  KeyPair(PrivateKey privateKey, PublicKey publicKey) {
+    if (privateKey == null || publicKey == null) {
```

Comment:
We can add a package-info.java file to the package and not have to deal with null values.

See https://github.com/ConsenSys/cava/blob/master/crypto/src/main/java/net/consensys/cava/crypto/package-info.java for an example

Reply:
Ok, so since there is already one in the crypto package I will just remove the null checks on private methods

Reply:
I am assuming you intended to point me to an example of a package-info.java that uses `@ParametersAreNonnullByDefault`.

Reply:
Yeah, and the crypto one should probably consider adding that! :)

Reply:
Interesting, I'll have to take a look